docs/learn/nexus-rules.md

```diff
@@ -0,0 +1,33 @@
+# Rules for storing data in NeXus
+
+!!! danger "Work in progress"
+
+There are several rules which apply for storing single data items in NeXus. There exists a [summary](https://manual.nexusformat.org/datarules.html) in the NeXus documentation outlining most of these rules. However, this explanation is not exhaustive and thus, we have compiled here additional information.
+
+
+## Namefitting
+
+In general, the names of NeXus group and field items are validated according to these boundaries:
```

Comment:
L10-22 looks like a copy from the above referenced NeXus documentation page. That would make sense to include references in these cases to the give subsection.
E.g.: NeXus documentation chapter "1.2.3.2. Rules for Storing Data Items in NeXus Files" section "NXDL group and field names" (https://manual.nexusformat.org/datarules.html)

Reply:
The idea was to state this here explicitly, so people don't have to search across multiple manuals. I think we can rewrite this a bit and make it even more clear, then we can have an individual statement. Note that the page is anyway linked.